cmd/syncthing/cli/show.go

```diff
@@ -40,5 +45,43 @@ var showCommand = cli.Command{
 			Usage:  "Show usage report",
 			Action: expects(0, dumpOutput("svc/report")),
 		},
+		{
+			Name:      "paused",
+			Usage:     "Show whether the folder/device is paused",
+			ArgsUsage: folderArgsUsage,
+			Action:    expects(2, paused),
+		},
 	},
 }
+
+func paused(c *cli.Context) error {
+	switch c.Args()[0] {
+	case "devices", "folders":
+	default:
+		return fmt.Errorf("first argument must be device or folder")
+	}
+	client := c.App.Metadata["client"].(*APIClient)
+	response, err := client.Get(strings.Join(append([]string{"config"}, c.Args()...), "/"))
+	if err != nil {
+		return err
+	}
+	bytes, err := responseToBArray(response)
+	if err != nil {
+		return err
+	}
+	var paused bool
+	if c.Args()[0] == "device" {
```

Comment:
This check doesn't work, it always treats the structure as a folder.  I wonder why `Unmarshal()` works, though?